zuul.d/projects.yaml

```diff
@@ -9,3 +9,4 @@
               - openstack-k8s-operators-content-provider
         - podified-multinode-edpm-deployment-crc: *content_provider
         - content-provider-data-plane-adoption-github-rdo-centos-9-extracted-crc: *content_provider
```

Comment:
i don't intend to merge this as is. I will update to remove the existing job from the layout here before we merge.

lets get at least a couple of runs of both for comparison/stability etc

Reply:
ok we've had 2 green runs now. latest there https://review.rdoproject.org/zuul/buildset/b6605e418ef046789c33b84799565aff

```
content-provider-data-plane-adoption-OSP-17-to-extracted-crc    1 hr 56 mins 51 secs SUCCESS
content-provider-data-plane-adoption-github-rdo-centos-9-extracted-crc    2 hrs 15 mins 24 secs SUCCESS
```

---

I sanity checked the new job is consuming the content provider meta operator image: https://logserver.rdoproject.org/53/653/01076f4673ef6ea1c92a0a40822d7620685fad1f/github-check/content-provider-data-plane-adoption-OSP-17-to-extracted-crc/34e317b/job-output.txt

```
2023-12-06 09:37:01.798666 | controller | ok: [localhost] => {"ansible_facts": {"cifmw_edpm_prepare_common_env": {"KUBECONFIG": "/home/zuul/.crc/machines/crc/kubeconfig", "NETWORK_MTU": "1500", "NNCP_DNS_SERVER": "192.168.122.10", "NNCP_INTERFACE": "ens7", "OUT": "/home/zuul/ci-framework-data/artifacts/manifests", "OUTPUT_DIR": "/home/zuul/ci-framework-data/artifacts/edpm", "PATH": "/home/zuul/.crc/bin:/home/zuul/.crc/bin/oc:/home/zuul/bin:/home/zuul/.local/bin:/home/zuul/bin:/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin"}, "cifmw_edpm_prepare_make_openstack_deploy_prep_env": "CLEANUP_DIR_CMD: \"true\"\nOPENSTACK_BRANCH: \"\"\nGIT_CLONE_OPTS: \"-l\"\nOPENSTACK_REPO: ~/src/github.com/openstack-k8s-operators/openstack-operator\n", "cifmw_edpm_prepare_make_openstack_env": "OPENSTACK_IMG: 38.129.56.112:5001/openstack-k8s-operators/openstack-operator-index:d12d9aa0db8cdff7943f921eaabf98171199f667\n", "cifmw_edpm_prepare_operators_build_output": {"openstack-operator": {"git_commit_hash": "d12d9aa0db8cdff7943f921eaabf98171199f667", "git_src_dir": "~/src/github.com/openstack-k8s-operators/openstack-operator", "image": "38.129.56.112:5001/openstack-k8s-operators/openstack-operator:d12d9aa0db8cdff7943f921eaabf98171199f667", "image_bundle": "38.129.56.112:5001/openstack-k8s-operators/openstack-operator-bundle:d12d9aa0db8cdff7943f921eaabf98171199f667", "image_catalog": "38.129.56.112:5001/openstack-k8s-operators/openstack-operator-index:d12d9aa0db8cdff7943f921eaabf98171199f667"}}}, "changed": false}
```

going to update and leave only the new job in the layout for merge